### PR TITLE
Added logging for where config logs and cache directories are stored

### PR DESCRIPTION
**Pull request by PeachesMLG**
Added logging for where config logs and cache directories are stored

**Changes**
Took me a while to figure out where the config directory was, so I thought some logging for this might help any future people having the same issue

**Issues**
N/A

### CODE REVIEW

**Comment by JPVenson** [comment]
FYI those paths are also shown in the Dashboard WebUI

image

**Comment by JPVenson** [comment]
Also, the Cache path should already be printed in the log by the BaseConfigurationManager

File: Jellyfin.Server/Helpers/StartupHelpers.cs [hunk 1]
```diff
@@ -57,6 +57,9 @@ public static void LogEnvironmentInfo(ILogger logger, IApplicationPaths appPaths
         logger.LogInformation("User Interactive: {IsUserInteractive}", Environment.UserInteractive);
         logger.LogInformation("Processor count: {ProcessorCount}", Environment.ProcessorCount);
         logger.LogInformation("Program data path: {ProgramDataPath}", appPaths.ProgramDataPath);
+        logger.LogInformation("Log Directory Path: {LogDirectoryPath}", appPaths.LogDirectoryPath);
+        logger.LogInformation("Config Directory Path: {ConfigurationDirectoryPath}", appPaths.ConfigurationDirectoryPath);
+        logger.LogInformation("Cache Path: {CachePath}", appPaths.CachePath);
```
Comment (Bond-009) [inline Jellyfin.Server/Helpers/StartupHelpers.cs:62]:
```suggestion
        logger.LogInformation("Log directory path: {LogDirectoryPath}", appPaths.LogDirectoryPath);
        logger.LogInformation("Config directory path: {ConfigurationDirectoryPath}", appPaths.ConfigurationDirectoryPath);
        logger.LogInformation("Cache path: {CachePath}", appPaths.CachePath);
```
Consistent with the other log lines

**Review by Bond-009 — CHANGES_REQUESTED** [review]

**Comment by PeachesMLG** [comment]
> FYI those paths are also shown in the Dashboard WebUI
> 
> image

Thats good to know actually, thanks

**Review by crobibero — APPROVED** [review]

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**